starcheck/plot.py

```diff
@@ -0,0 +1,187 @@
+import os
+import logging
+
+import Quaternion
+import numpy as np
+import agasc
+from Chandra.Time import DateTime
+import matplotlib.pyplot as plt
+from Ska.quatutil import radec2yagzag
+import Ska.Numpy
+from astropy.table import Table
+
+TASK_NAME = 'starcheck_plotting'
+logger = logging.getLogger(TASK_NAME)
+
+# rc definitions
+frontcolor = 'black'
+backcolor = 'white'
+plt.rcParams['lines.color'] = frontcolor
+plt.rcParams['patch.edgecolor'] = frontcolor
+plt.rcParams['text.color'] = frontcolor
+plt.rcParams['axes.facecolor'] = backcolor
+plt.rcParams['axes.edgecolor'] = frontcolor
+plt.rcParams['axes.labelcolor'] = frontcolor
+plt.rcParams['xtick.color'] = frontcolor
+plt.rcParams['ytick.color'] = frontcolor
+plt.rcParams['grid.color'] = frontcolor
+plt.rcParams['figure.facecolor'] = backcolor
+plt.rcParams['figure.edgecolor'] = backcolor
+plt.rcParams['savefig.facecolor'] = backcolor
+plt.rcParams['savefig.edgecolor'] = backcolor
+
+
+def symsize(mag):
+    # map mags to figsizes, defining
+    # mag 6 as 20 and mag 11 as 1
+    # interp should leave it at the bounding value outside
+    # the range
+    return np.interp(mag, [6.0, 11.0], [20.0, 1])
```

Comment:
Might be simplest to revert the change I added for a different figure sizes and just image resize the field plot.